onyo/commands/set.py

```diff
@@ -166,6 +166,7 @@ def prepare_arguments(source, keys, quiet, yes, recursive, depth, onyo_root):
     if problem_str != "":
         logger.error(problem_str)
         sys.exit(1)
+    asset_list.sort()
```

Comment:
Just so I understand this correctly:

If I run `onyo cat b a` it will list the output of `a` first and then `b`?

If so, I would feel more comfortable if this change came with a discussion (or at least opening an issue in which we can discuss this behavior, and merge this for now).

IMO, the order should be predictable, and as the user (or test) specified.

Reply:
This function is just for set, cat still has input order.

The issue was opened yesterday as #135.